.github/workflows/ci-workflow.yaml

```
@@ -0,0 +1,23 @@
+name: CI Workflow 
+on: [push, pull_request]
+jobs:
+  build:
+    name: build and test
+    runs-on: ubuntu-latest
+    strategy:
```

```suggestion
    strategy:
      fail-fast: false
```

Am a fan of disabling the [`fail-fast`](https://help.github.com/en/github/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast) strategy, to let the full workflow run even when something fails early. Suppose it's up for debate.

---

what's the use case here? suppose we have test cases are running and if any of the test cases fails then CI should fail, correct me if I am wrong here

---

I'm okay with this too so I'd add it.

---

@Lobarr It's open to discussion, you don't necessarily have to add it

---

This is helpful with the `matrix` strategy, say the version matrix is like
```
        node-version: [9.x, 10.x, 11.x]
```

When `fail-fast` is `true`, say anything fails in the `9.x` builds - it cancels all the other builds. But we're still interested in the results for other versions, no? Disabling `fail-fast` continues all in-progress jobs if any `matrix` job fails.

---

Ohh! Gotcha - Thanks 👍